### ISSUE TITLE
Email to/from headers showing encoded string

### ISSUE THREAD

**Issue by somiaj** [openwebwork/webwork2#1667]
I was testing sending emails and the email headers I got looked like:

```
from: =?UTF-8?B?SmFpbW9zIFNrcmlsZXR6?= <name@domain.com>
to: =?UTF-8?B?SmFpbW9zIFNrcmlsZXR6?= <name@domain.com>
```

Looking into this it appears that Email::Stuffer uses `header_str_set` to set the header strings, which will encode strings that contain control or 8-bit characters.

https://metacpan.org/pod/Email::MIME#header_str_set

I believe this means that the `rfc822_mailbox` method in `lib/WeBWorK/DB/Record/User.pm` no longer needs to encode `$full_name` since Email::Stuffer will do the encoding if needed.

**Comment by drgrice1** [comment]
I have a branch that already does this.  When I switched to using Email::Stuffer I planned on  doing this.  In addition to the minor issue on the email page that you observed there is a more serious issue.  It actually raises the spam score of emails to unnecessarily base64 encode an address.  An email sent from webwork comes in with something like this in it when it goes through the pipeline of my universities email server:
```
X-Barracuda-Spam-Report: Code version 3.2, rules version 3.2.3.97391
	Rule breakdown below
	 pts rule name              description
	---- ---------------------- --------------------------------------------------
	0.14 MISSING_MID            Missing Message-Id: header
	0.01 FROM_EXCESS_BASE64     From: base64 encoded unnecessarily
	1.05 FROM_EXCESS_BASE64_2   From: base64 encoded unnecessarily
```
The missing message id is something I haven't figured out what to do about yet, but the other two are caused by this issue.

**Comment by somiaj** [comment]
@drgrice1 The message ID might be an issue of the SMTP server being used and not WeBWorK. I have tried to look into this and found a stackexchange post that talks about how the server should be adding message IDs to the emails it sends out. At my university I use a mail relay, and when I look at the headers of the emails sent by WeBWorK, they correctly have a Message-Id included in the email header from the SMTP relay I am using.

**Comment by drgrice1** [comment]
That probably is the case.  So that is not WeBWorK's problem.

**Comment by pstaabp** [comment]
This was fixed in #1668.
